webroot/src/store/user/user.state.ts

```diff
@@ -20,7 +20,7 @@ export interface State {
 
 export const state: State = {
     model: null,
-    isLoggedIn: (authStorage.getItem(tokens.staff.AUTH_TOKEN) !== null),
+    isLoggedIn: (!!authStorage.getItem(tokens.staff.AUTH_TOKEN) || !!authStorage.getItem(tokens.licensee.AUTH_TOKEN)),
```

Comment:
What changed that requires the generic `!!` vs an explicit check against `null`?

Reply:
The change is two things:
1) We now have two conditions on one line and this notation is slightly more concise
2) This is a little more internally resilient. If we get anything falsy from those expressions we should not be considered logged in. Because of the way the storage libraries are set up and the storage method's nature Im not immediately seeing an instance where these two things aren't identical. Generally though, decoupling and decoupling resiliency is a good thing. So in my mind this is either a small plus or a nothing.